### ISSUE TITLE
Can't sign in to mods.php

### ISSUE THREAD

**Issue by c0mp1e-x** [lainchan/lainchan#138]
When I try and log in it returns the error "Database error: SQLSTATE[HY000] [1045] Access denied for user 'hidden'@'localhost' (using password: YES)"

So far I've tried using the default username and password, I've also tried changing the mods username/password in the MySQL database I selected during the installation.

**Comment by AshtonHarding** [comment]
Try `127.0.0.1` instead of `localhost`

**Comment by c0mp1e-x** [comment]
@AshtonHarding I've tried that now, I'm still getting the same error. I'm not sure what to do from here. Maybe I messed up during installation or while editing a SQL database.

**Comment by AshtonHarding** [comment]
Damn. Ok. I'd say try to reformat the sql db. If that doesn't work, you could try to find out vagrant we have somewhere floating around. I can't remember where it's at. But if you asked in irc (#programming or #thecage) -- Someone should be able to tell you.

**Comment by c0mp1e-x** [comment]
@AshtonHarding so I just decided to fresh install the whole thing which only took 15 minutes, I set the database to mysql when I installed it. I'm not sure exactly what to do to set an admin username and password, I was pretty sure I did it right before but considering it didn't work I'd prefer to ask first to make sure I do it right because I'm still pretty new to sql. Anything I try logging in with at the moment returns the same error as it did before.
